Gemfile

```diff
@@ -1,5 +1,4 @@
 source 'https://rubygems.org'
 
-gem 'jekyll',     '=0.11.0'
-gem 'liquid',     '=2.2.2'
-gem 'RedCloth',   '=4.2.9'
+gem 'github-pages'
+gem 'kramdown'
```

Comment:
Ha! Because @jimmynotjim told me yesterday, and I quote:

> when you redo that PR, make sure to add gem "kramdown" to the gemfile
> same as the PR I made to the design manual

:stuck_out_tongue: Backing it out.